projects/igniteui-angular/migrations/common/tsUtils.ts

```diff
@@ -45,7 +50,9 @@ export const getIdentifierPositions = (source: string | ts.SourceFile, name: str
                 return false;
             }
         }
-        return node.text === name;
+        // for methods the node.text will not contain characters like ()
+        const cleanName = name.match(/\w+/g)[0] || name;
```

Comment:
@jackofdiamond5 for future improvement: this can probably be resolved with some `kind` check and likely the method nodes will have `name` or something